src/Common/Simulation.cpp

```diff
@@ -188,6 +188,7 @@ SimulationBase::SimulationBase
 #else
   Nthreads = 1;
 #endif
+  timing = new CodeTiming() ;
```

Comment:
Well, ok, I accept this requires fewer modications the code - let's leave it like this